lib/arrow_payments.rb

```diff
@@ -1,24 +1,25 @@
-require 'arrow_payments/version'
-require 'arrow_payments/errors'
+require "arrow_payments/version"
+require "arrow_payments/errors"
 
 module ArrowPayments
-  autoload :Configuration,    'arrow_payments/configuration'
-  autoload :Entity,           'arrow_payments/entity'
-  autoload :Customer,         'arrow_payments/customer'
-  autoload :PaymentMethod,    'arrow_payments/payment_method'
-  autoload :RecurringBilling, 'arrow_payments/recurring_billing'
-  autoload :Transaction,      'arrow_payments/transaction'
-  autoload :Address,          'arrow_payments/address'
-  autoload :LineItem,         'arrow_payments/line_item'
-  autoload :Connection,       'arrow_payments/connection'
-  autoload :Client,           'arrow_payments/client'
+  autoload :Configuration,    "arrow_payments/configuration"
```

Comment:
I'm getting an ArgumentError if I remove the second argument. It looks like in ruby 2.3.x it's still required.

http://ruby-doc.org/core-2.3.0/Kernel.html#method-i-autoload